src/keyfile.c

```diff
@@ -86,12 +86,10 @@
 #define GEANY_DEFAULT_TOOLS_BROWSER		"open -a safari"
 #define GEANY_DEFAULT_FONT_SYMBOL_LIST	"Helvetica Medium 12"
 #define GEANY_DEFAULT_FONT_MSG_WINDOW	"Menlo Medium 12"
-#define GEANY_DEFAULT_FONT_EDITOR		"Menlo Medium 12"
```

Comment:
@techee for this we could use "Vendor Overrides" mentioned [in the API docs](https://developer.gnome.org/gio/stable/GSettings.html#GSettings.description).